views/partials/scan_form.hbs

```diff
@@ -3,12 +3,12 @@
       Please enter a valid email.
     </span>
   <div class="input-group">
-    <input class="input-group-field email-to-hash" type="email" name="email" placeholder="Enter Email Address" aria-label="Enter Email Address" />
+    <input class="input-group-field email-to-hash" type="email" name="email" placeholder="Enter Email Address" aria-label="Enter Email Address" tabindex="0" maxlength="80"/>
```

Comment:
Why an email address length limit of 80 specifically? Is that a DB limitation, or was allowing longer strings causing some other problem?

Reply:
It's not a DB limitation. Why are we doing this indeed? :)